tests/neptune/new/attributes/atoms/test_artifact_hash.py

```diff
@@ -31,6 +31,17 @@ def test_assign_type_error(self):
             with self.assertRaises(Exception):
                 Artifact(MagicMock(), MagicMock()).assign(value)
 
+    def test_fetch(self):
+        exp, path = self._create_run(), self._random_path()
+        var = Artifact(exp, path)
+        var._enqueue_operation(
+            AssignArtifact(
+                var._path,
+                ArtifactVal("e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855").hash
+            ),
+            False)
```

Comment:
I didn't check it but if this argument is `wait` maybe it should be set to `True`?

Reply:
it's setup using mock backend, so meh